core/src/cubos/core/memory/stream.cpp

```diff
@@ -125,11 +126,11 @@ void Stream::print(double value, size_t decimalPlaces)
     // 16777217 is the last number whose integer part can be represented exactly
     else if (value <= 16777217.0 && log10(value) > -static_cast<double>(decimalPlaces))
     {
-        char buffer[MAX_DECIMAL_PLACES + 1];
+        std::array<char, MAX_DECIMAL_PLACES + 1> buffer;
         size_t len = 0;
 
         // Print the integer part
-        uint64_t integerPart = static_cast<uint64_t>(value);
+        auto integerPart = static_cast<uint64_t>(value);
```

Comment:
*ditto* (and for all other 3 cases as well)
```suggestion
        uint64_t integerPart = static_cast<uint64_t>(value);
```